include/rdma/fabric.h

```diff
@@ -121,6 +121,7 @@ typedef struct fid *fid_t;
 #define FI_TRANSMIT_COMPLETE	(1ULL << 27)
 #define FI_DELIVERY_COMPLETE	(1ULL << 28)
 
+#define FI_SELECTIVE_COMPLETION	(1ULL << 55)
```

Comment:
This is a bind specific call flag, but it is being grouped with the fi_getinfo specific caps.  Please move the definition closer to struct fi_ops, where bind is defined, and change the value to 59.  FI_SELECTIVE_COMPLETE is not used with fi_getinfo.